src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,12 @@
 package lotto;
 
+import lotto.controller.LottoController;
+import lotto.domain.LotteryMachine;
+import lotto.view.InputViewImpl;
+import lotto.view.OutputViewImpl;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        new LottoController(new InputViewImpl(), new OutputViewImpl(), new LotteryMachine()).run();
```

Comment:
줄바꿈을 사용해서 인자 가독성을 높일 수 있을 것 같아요 ~!
```suggestion
        new LottoController(
                new InputViewImpl(),
                new OutputViewImpl(),
                new LotteryMachine()
                ).run()
```